### PR TITLE
create sub resource supporting CRUDL and add tests

### PR DESCRIPTION
**Pull request by jxu577**
cc @kyleconroy @cupcait

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -161,6 +161,12 @@ stripe.balance.retrieve({
   * [`update(skuId[, params])`](https://stripe.com/docs/api/node#update_sku)
   * [`retrieve(skuId)`](https://stripe.com/docs/api/node#retrieve_sku)
   * [`del(skuId)`](https://stripe.com/docs/api/node#delete_sku)
+ * subscriptions
+  * [`create(params)`](https://stripe.com/docs/api/node#create_subscription_beta)
+  * [`list([params])`](https://stripe.com/docs/api/node#list_subscription_beta)
+  * [`update(skuId[, params])`](https://stripe.com/docs/api/node#update_subscription_beta)
+  * [`retrieve(skuId)`](https://stripe.com/docs/api/node#retrieve_subscription_beta)
```
Comment (matthewarkin) [inline README.md]: /s/skuId/subId?

Comment (jxu577) [inline README.md]: whoops copy pasta

**Comment by cupcait** [comment]
👍

**Comment by cupcait** [comment]
cc @stripe/api-libraries

**Comment by kyleconroy** [comment]
We'll need to update the docs for the `del` method. It's currently documented as `stripe.subscriptions.delete`.

**Comment by kyleconroy** [comment]
:+1: This is good to go

**Pull request merged**